load.php

```diff
@@ -477,17 +477,37 @@ function perflab_run_module_activation_deactivation( $old_value, $value ) {
 				return;
 			}
 
-			$current_user = get_current_user_id();
-			$dismissed    = array_filter( explode( ',', (string) get_user_meta( $current_user, 'dismissed_wp_pointers', true ) ) );
-
-			if ( ! in_array( 'perflab-module-migration-pointer', $dismissed, true ) ) {
-				return;
+			$current_user       = wp_get_current_user();
+			$current_user_roles = array_shift( $current_user->roles );
+
+			$args = array(
+				'role'       => $current_user_roles,
+				'meta_query' => array(
+					array(
+						'key'     => 'dismissed_wp_pointers',
+						'value'   => 'perflab-module-migration-pointer',
+						'compare' => 'LIKE',
+					),
+				),
```

Comment:
We may want to use `get_users()` instead of manually relying on `WP_User_Query`. It makes the code a bit simpler and also automatically sets `count_total` to `false`, which makes the query faster.

Reply:
Updated in https://github.com/WordPress/performance/pull/917/commits/f4fcc72de9bf6475098be8920f942e8ef7c21872